data/sdo-howto-examples.txt

```diff
@@ -92,98 +92,342 @@ PRE-MARKUP:
-      </div>
-    </div>
-  </div>
-  <div itemprop="step" itemscope itemtype="http://schema.org/HowToSection">
-    <div itemprop="name">Finishing up</div>
-    <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToStep">
-      <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToDirection">
-        <div itemprop="text">Lower the jack and tighten the lug nuts with the wrench.</div>
-      </div>
-    </div>
-    <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToStep">
-      <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToDirection">
-        <div itemprop="text">Replace the hubcap.</div>
-      </div>
-    </div>
-    <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToStep">
-      <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToDirection">
-        <div itemprop="text">Put the equipment and the flat tire away.</div>
-      </div>
+    <strong><span itemprop="name">Change a Flat Tire</span></strong>
+    <div>About <span itemprop="estimatedCost" itemscope itemtype="http://schema.org/MonetaryAmount">
+        <meta itemprop="currency" content="USD"></meta>
+        <meta itemprop="value" content="20"></meta>
+    </span>$20</div>
+    <div>About <span itemprop="totalTime" content="PT30M">30 minutes</span></div>
+    <div>Necessary Items:</div>
+    <div itemprop="tool" itemscope itemtype="http://schema.org/HowToTool">Spare tire</div>
+    <div itemprop="tool" itemscope itemtype="http://schema.org/HowToTool">Lug wrench</div>
+    <div itemprop="tool" itemscope itemtype="http://schema.org/HowToTool">Jack</div>
+    <div itemprop="tool" itemscope itemtype="http://schema.org/HowToTool">Wheel wedges</div>
+    <div itemprop="supply" itemscope itemtype="http://schema.org/HowToSupply">Flares</div>
+    <div itemprop="steps" itemscope itemtype="http://schema.org/ItemList">
+        <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToSection">
+            <div itemprop="name">Preparation</div>
+            <meta itemprop="position" content="1"></meta>
+            <div itemprop="steps" itemscope itemtype="http://schema.org/HowToStep">
+                <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToDirection">
+                    <meta itemprop="position" content="1"></meta>
+                    <img itemprop="duringMedia" href="lit-flares.jpg" />
+                    <div itemprop="text">Turn on your hazard lights and set the flares.</div>
+                </div>
+                <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToTip">
+                    <meta itemprop="position" content="2"></meta>
+                    <div itemprop="text">You're going to need space and want to be visible.</div>
+                </div>
+                <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToDirection">
```

Comment:
I believe this changes the meaning of the HowTo Steps. Originally there was a HowToStep wrapping each HowToDirection. My understanding is that a HowToStep represents a (usually numbered) significant step, which can contain the HowToDirection primary text and optionally a HowToTip (a side note for more complex steps). 

Was there any intent / discussion around changing the design of the HowTo for this release? If not, I'd suggest keeping the original HowToStep structure (wrapping the direction + tip) for now.